modules/openapi-generator/src/main/resources/scala-play-framework/simpleParentJsonFormat.mustache

```diff
@@ -0,0 +1,7 @@
+{{! Keep indentation at 4 spaces for proper formatting of the output. }}
```

Comment:
I've found mustache to be inconsistent with indents in some situations. I don't remember the specifics, but I think indents may be dropped in files at an import depth greater than 1?

To resolve the issue I had created a lambda that you can reuse if the indents become an issue. See the kotlin-server generator. Example usage of the lambda: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/_api_body.mustache

Reply:
Thanks! I'll try experimenting more with the indents and if nothing works I'll give this method a shot.